client/signup/controller.js

```diff
@@ -353,12 +328,34 @@ export default {
 			context.store.dispatch( updateDependencies( additionalDependencies ) );
 		}
 
+		/**
+		 * The experiment is only loaded on the onboarding flow
+		 * If user is logged out we load the experiment
+		 * If user is logged in we load the experiment only if the user has no sites
+		 * More info: pbxNRc-3xO-p2
+		 */
+		const isNewUser = ! getCurrentUserSiteCount( context.store.getState() );
```

Comment:
Any reason why we want to restrict the survey to only those with zero sites? A note that those with zero sites may not necessarily be a new user.